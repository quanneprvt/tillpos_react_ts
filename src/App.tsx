import React from "react";
import Home from "./views/Home";
import "./App.css";

class App extends React.Component {
  render() {
    return <Home />;
  }
}

export default App;
