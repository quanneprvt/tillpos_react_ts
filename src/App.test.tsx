import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';
import App from './App';

describe("APP View", () => {
  let wrapper: ShallowWrapper;
  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it("should render correctly", () => expect(wrapper).toMatchSnapshot());
});
