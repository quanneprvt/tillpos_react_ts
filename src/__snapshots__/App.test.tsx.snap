// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APP View should render correctly 1`] = `ShallowWrapper {}`;
